src/hypothesis/internal/charmap.py

```diff
@@ -141,6 +141,40 @@ def _union_intervals(x, y):
     return tuple(result)
 
 
+def _subtract_intervals(x, y):
+    """Return a list of intervals that bounds everything bounded by x that is
```

Comment:
Consistency nit: everything else uses n-tuples of (int, int) for the intervals type; I'd do the same here unless there's specific reason not to.